xtext/org.icyphy.linguafranca/src/org/icyphy/generator/GeneratorBase.xtend

```diff
@@ -984,6 +992,30 @@ abstract class GeneratorBase extends AbstractLinguaFrancaValidator {
         }
     }
     
+    /**
+     * Run the custom build command specified in "build" parameter
+     */
+    protected def runBuildCommand() {
+	var _buildCommand = newArrayList('-c', targetBuildCommand)
+        val buildCommand = createCommand("bash", _buildCommand)
```

Comment:
I see why you are doing this (`createCommand` requires a separate command + args), but now we need bash in order to execute a build command. That doesn't seem right to me.